site/includes/footer.pug

```diff
@@ -1,8 +1,11 @@
-.spectrum-Site-footerContainer
-  hr.spectrum-Rule.spectrum-Rule--small
+unless process.env.VRT
 
-  div(id="footer_global")
+  .spectrum-Site-footerContainer
 
-  include adobe-globalnav.js
+    hr.spectrum-Rule.spectrum-Rule--small
 
-  script(src="//wwwimages2.adobe.com/etc/beagle/public/globalnav/adobe-globalnav/latest/adobe-globalnav.min.js")
+    div(id="footer_global")
+
+    include adobe-globalnav.js
+
+    script(src="//wwwimages2.adobe.com/etc/beagle/public/globalnav/adobe-globalnav/latest/adobe-globalnav.min.js")
```

Comment:
see above

Reply:
Same as above. Enable VRT=true, this script tag will not be rendered.